packages/vue/src/components/pages/category/Category.vue

```diff
@@ -515,6 +517,28 @@ export default {
           rating: { max: 5, score: 5 },
           reviewsCount: 8,
           isInWishlist: true,
+          colors: [
+            { label: "Sand", value: "sand", color: "#EDCBB9", selected: false },
+            { label: "Mint", value: "mint", color: "#ABD9D8", selected: false },
+            {
+              label: "Vivid rose",
+              value: "vivid rose",
+              color: "#DB5593",
+              selected: false,
+            },
+            {
+              label: "Peach",
+              value: "peach",
+              color: "#F59F93",
+              selected: false,
+            },
+            {
+              label: "Citrus",
+              value: "citrus",
+              color: "#FFEE97",
+              selected: false,
+            },
+          ],
```

Comment:
What do you think about adding a separate file for this data and importing them, like `internalData.js` file in templates?

Reply:
I can add a separate file but I'm wondering if we want to show it in source code :) what do you think?

Reply:
You are right, let's stay with this :)